src/main.rs

```diff
@@ -182,22 +192,40 @@ async fn main() {
     axum::serve(listener, app).await.unwrap();
 }
 
+
+//ToString -> RTCSessionDescription
 async fn webrtc_handler(
     State(state): State<SharedState>,
-    Json(offer): Json<RTCSessionDescription>,
-) -> Json<RTCSessionDescription> {
-    let (answer, sender) = start_webrtc(offer).await;
+    request:Request,
+) -> 
+   // Result<(),(StatusCode,String)>
+   impl IntoResponse
+       {
+    //body
+    let body=request.into_body();
+    //bytes
+    let body_bytes = hyper::body::to_bytes(body).await.unwrap();
+    //String
```

Comment:
Remove type comment, the type should auto show in your editor

For example: vscode

![image](https://github.com/binbat/live777/assets/18189138/4915efb4-2fa7-45d5-8e49-23b9f8efb401)


https://code.visualstudio.com/docs/languages/rust

Reply:
ok